lib/CachePlugin.js

```diff
@@ -6,18 +6,40 @@
 
 const asyncLib = require("neo-async");
 
+/** @typedef {import("./Compiler.js")} Compiler */
+/** @typedef {import("./MultiCompiler.js")} MultiCompiler */
```

Comment:
These 2 imports are no longer needed

Reply:
A yes, totally forgot the remove them. thanks..